apps/storefront/src/app.ts

```diff
@@ -17,4 +17,7 @@ export const app = appBuilder()
   .withFeature(features)
   .withTheme([storefrontTheme])
   .withEnvironment(env)
+  .withOptions({
+    cart: { multi: env.ORYX_PRESET === 'b2b' || env.ORYX_MULTI_CART },
```

Comment:
Shouldn't it be on a `presets` feature level? Especially keeping in mind enabling `ORYX_MULTI_CART` can't enable this feature by itself.

Reply:
Dropped for current implementation